src/locales/en-US/commands.json

```diff
@@ -42,10 +42,70 @@
     "commandDescription": "Checks if the bot is online"
   },
   "spotify": {
-    "commandDescription": "Searches for a song on Spotify",
-    "commandUsage": "<track name>",
+    "commandDescription": "Searches for a song, album, artist or a playlist and displays an user info on Spotify",
+    "commandUsage": "<type> <query>",
     "noTracksFound": "I couldn't find any tracks with that name.",
-    "noTrackName": "You need to give me a track name."
+    "noType": "Please insert a valid type",
+    "types": "Types",
+    "album": "Album",
+    "genres": "Genres",
+    "artist": "Artist",
+    "artistPlural": "Artists",
+    "track": "Track",
+    "trackPlural": "Tracks",
+    "albumPlural": "Albums",
+    "moreTracks": "...and more {{tracks}} tracks",
+    "moreAlbums": "...and more {{albums}} albums",
+    "followers": "Followers",
+    "followersCount": "{{followers}} followers",
+    "selectResult": "Please send the number of the result (10 seconds)",
+    "subcommands": {
+      "track": {
+        "commandDescription": "Searches for a song on Spotify",
+        "commandUsage": "<song>",
+        "noTrack": "You need to give me a track name.",
+        "notFound": "I couldn't find any track with that name.",
+        "results": "Track results for {{query}}",
+        "trackInfo": "Track information"
+      },
+      "album": {
+        "commandDescription": "Searches for an album on Spotify",
+        "commandUsage": "<album>",
+        "noAlbum": "You need to give me an album name.",
+        "notFound": "I couldn't find any album with that name.",
+        "results": "Album results for {{query}}",
+        "albumInfo": "Album information",
+        "albumType": "Album type",
+        "types": {
+          "single": "Single",
+          "compilation": "Compilation"
+        }
+      },
+      "artist": {
+        "commandDescription": "Searches for an artist on Spotify",
+        "commandUsage": "<artist>",
+        "noArtist": "You need to give me an artist name.",
+        "notFound": "I couldn't find any artist with that name.",
+        "results": "Artist results for {{query}}",
+        "artistInfo": "Artist information"
+      },
+      "playlist": {
+        "commandDescription": "Searches for a playlist on Spotify",
+        "commandUsage": "<playlist>",
+        "noPlaylist": "You need to give me a playlist name.",
+        "notFound": "I couldn't find any playlist with that name.",
+        "results": "Playlist results for {{query}}",
+        "playlistInfo": "Playlist information",
+        "createdBy": "Created by"
```

Comment:
Is it possible to add the creator to the string? It would be way easier for us to translate it.

Reply:
look the screenshots